build-scripts/build_compiler.js

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2018 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Build the compiler java jar and GWT platforms from source.
+ *
+ * Invoked as part of the package build process:
+ *
+ *     yarn run build
+ */
+'use strict';
+
+const {spawn, spawnSync} = require('child_process');
+const ncp = require('ncp');
+const Semver = require('semver');
+
+// Master version number of the packages. Replaces the version
+// information from the root package.json file.
+const packageInfo = require('../lerna.json');
+const packageVersion = new Semver(packageInfo.version);
+
+const fs = require('fs');
+const glob = require('glob');
+
+const gitTagResults = spawnSync('git', ['tag', '--points-at', 'HEAD'], {
```

Comment:
Could you create a function named something like `getCompilerSubmoduleReleaseVersion()` to make it clearer what's going on here?